### PR TITLE
Link to the place where signature options are defined

### PR DESCRIPTION
**Pull request by tomato42**
finding the name of `nonce-type` to know that's the name to use deterministic ECDSA is quite frustrating otherwise...

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

**Review by beldmit — APPROVED** [review]
LGTM

**Comment by beldmit** [comment]
I think it's relevant for all 3.x branches but not sure if we backport doc fixes

**Comment by tomato42** [comment]
it's not relevant for 3.1 and earlier as there's no deterministic (EC)DSA there

**Comment by beldmit** [comment]
The signature options are still documented in `provider-signature`, I think

**Comment by tomato42** [comment]
:woman_facepalming: right, yes, it's still useful for stuff like RSA-PSS

**Review by nhorman — APPROVED** [review]

File: doc/man1/openssl-ca.pod.in [hunk 1]
```diff
@@ -179,7 +179,8 @@ See L<openssl-format-options(1)> for details.
 =item B<-sigopt> I<nm>:I<v>
 
 Pass options to the signature algorithm during sign operations.
-Names and values of these options are algorithm-specific.
+Names and values of these options are algorithm-specific and
+documented in L<provider-signature(7)>.
```
Comment (slontis) [inline doc/man1/openssl-ca.pod.in:183]: /Signature parameters> ?

Comment (tomato42) [reply doc/man1/openssl-ca.pod.in:183]: sorry, don't understand the request...

Comment (slontis) [reply doc/man1/openssl-ca.pod.in:183]: I was suggesting changing the link to include the section within the provider-signature document.

Comment (tomato42) [reply doc/man1/openssl-ca.pod.in]: fixed

Comment (slontis) [reply doc/man1/openssl-ca.pod.in]: May as well change the other file also..

Comment (tomato42) [reply doc/man1/openssl-ca.pod.in]: done

**Review by t8m — APPROVED** [review]
Yes, we do backport relevant doc fixes.

**Comment by t8m** [comment]
@nhorman please reconfirm

**Comment by nhorman** [comment]
ack, approval holds

**Comment by nhorman** [comment]
Ack, approval holds

**Review by romen — APPROVED** [review]
LGTM

**Comment by t8m** [comment]
Squashed and merged to the master, 3.3 and 3.2 branches. Thank you for your contribution.

**Pull request closed without merging**